general/device.cpp

```diff
@@ -383,6 +383,8 @@ static void RajaDeviceSetup(const int dev, int &ngpu)
 {
 #ifdef MFEM_USE_CUDA
    if (ngpu <= 0) { DeviceSetup(dev, ngpu); }
+#elif defined(MFEM_USE_HIP)
+   HipDeviceSetup(dev, ngpu);
```

Comment:
@camierjs would this work for RAJA as well?

Reply:
yes!